packages/types/typedoc.config.cjs

```diff
@@ -0,0 +1,35 @@
+const fs = require('node:fs');
+const path = require('node:path');
+
+function* walkSync(dir) {
+	const files = fs.readdirSync(dir, { withFileTypes: true });
+
+	for (const file of files) {
+		if (file.isDirectory()) {
+			yield* walkSync(path.join(dir, file.name));
```

Comment:
Any way we can put this in a separate package so we can reuse it in other packages as well? 🤔

Reply:
I'm afraid this won't work since we have to use CJS for this config 😢

Reply:
Oh you meant the whole config?